script/upgrades/MINIDROP/data/generate-airdrop-csv.ts

```diff
@@ -19,79 +19,106 @@ interface Airdrop {
 
 config();
 
+const MENTO_THRESHOLD = new Decimal(1e18);
+
 async function run() {
   const duneApiKey = process.env.DUNE_API_KEY;
-  const filepath = path.resolve(__dirname, "airdrop.csv");
-  clearFileContents(filepath);
+  if (!duneApiKey) {
+    console.error("🚨 Missing DUNE_API_KEY environment variable");
+    process.exit(1);
+  }
+
+  const mentoFilepath = path.resolve(__dirname, "mento_airdrop.csv");
+  const cUSDFilepath = path.resolve(__dirname, "cusd_airdrop.csv");
+  clearFileContents(mentoFilepath);
+  clearFileContents(cUSDFilepath);
 
-  const csvWriter = createObjectCsvWriter({
-    path: filepath,
+  const mentoCsvWriter = createObjectCsvWriter({
+    path: mentoFilepath,
     header: [
       { id: "address", title: "Address" },
       { id: "mentoAllocation", title: "MENTO Allocation" },
+    ],
+    append: true,
+  });
+
+  const cUSDCsvWriter = createObjectCsvWriter({
+    path: cUSDFilepath,
+    header: [
+      { id: "address", title: "Address" },
       { id: "cUSDAllocation", title: "cUSD Allocation" },
     ],
-    append: false,
+    append: true,
   });
 
   const queryId = 3932204; // https://dune.com/queries/3932204
```

Comment:
nit: would pull this magic hardcoded const up next to the MENTO_THRESHOLD constant on top